src/index.js

```diff
@@ -1,139 +1,128 @@
-if (config.type === 'browserify') {
-  // Normalize all require function calls to all contain the module id.
-  // var a = require('a') => var a = require(1)
-  const browserifyDecoder = require('./decoders/browserify');
-  modules = browserifyDecoder(iifeModules);
-} else {
-  const webpackDecoder = require('./decoders/webpack');
-  modules = webpackDecoder(iifeModules, config.knownPaths);
-}
+    console.log('* Decoding modules...');
 
+    let modules;
+    if (config.type === 'browserify') {
+        // Normalize all require function calls to all contain the module id.
+        // var a = require('a') => var a = require(1)
+        const browserifyDecoder = require('./decoders/browserify');
+        modules = browserifyDecoder(iifeModules);
+    } else {
+        const webpackDecoder = require('./decoders/webpack');
+        modules = webpackDecoder(iifeModules, config.knownPaths);
+    }
 
 
 // ------------------------------------------------------------------------------
 // Transform the module id in each require call into a relative path to the module.
 // var a = require(1) => var a = require('./path/to/a')
 // ------------------------------------------------------------------------------
 
-console.log('* Reassembling requires...');
-const transformRequires = require('./transformRequires');
-modules = transformRequires(modules, config.knownPaths, config.entryPoint, config.type, config.replaceRequires);
+    console.log('* Reassembling requires...');
+    const transformRequires = require('./transformRequires');
+    modules = transformRequires(modules, config.knownPaths, config.entryPoint, config.type, config.replaceRequires);
 
 // ------------------------------------------------------------------------------
 // Take the array of modules and figure out where to put each module on disk.
 // module 1 => ./dist/path/to/a.js
 // ------------------------------------------------------------------------------
 
-console.log('* Resolving files...');
-const lookupTableResolver = require('./lookupTable');
-const files = lookupTableResolver(
-  modules,
-  config.knownPaths,
-  config.entryPoint,
-  config.type,
-  outputLocation
-);
+    console.log('* Resolving files...');
+    const lookupTableResolver = require('./lookupTable');
+    const files = lookupTableResolver(
+        modules,
+        config.knownPaths,
+        config.entryPoint,
+        config.type,
+        outputLocation
+    );
 
 // ------------------------------------------------------------------------------
 // Finally, write the bundle to disk in the specified output location.
 // ------------------------------------------------------------------------------
 
-console.log('* Writing to disk...');
-const writeToDisk = require('./writeToDisk');
-writeToDisk(files);
+    console.log('* Writing to disk...');
+    const writeToDisk = require('./writeToDisk');
+    writeToDisk(files);
+}
+
+function debundle_file_inputs(bundleLocation,outputLocation,configPath)
+{
```

Comment:
Could you please stick to the conventions of the project and put curly braces at the end of the line? Also, `camelCase`.

ie:
```
function foo(a, b, c) {
  // code
}
```